### PR TITLE
Add admin's UI for object-store table

### PR DESCRIPTION
**Pull request by darkdarkdragon**
**What does this pull request do? Explain your changes. (required)**
UI for object store table

**Specific updates (required)**

## -

- **How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Screenshots (optional):**
image

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by wohlner** [comment]
Hold on. What is this? Can you please describe this UI feature in more detail. I want to keep informed of additions to our Livepeer.com admin and dashboard UIs. Thank you.

**Comment by darkdarkdragon** [comment]
@wohlner 
In our database we have table called 'object_store'. It contains credentials for the Google Storage buckets used by API to store recordings. I've added UI for it so I don't need to modify db directly.

**Comment by wohlner** [comment]
@darkdarkdragon Good thinking building tooling to avoid modifying the db directly. In the future, please give me a heads up before you new features like this to the admin. In a couple weeks, I am going to start on a project with Basement studios to revamp the livepeer.com Admin and Dashboard. The goal for the remodeled admin will be to make it super easy to manage Liveper.com users, streams and API keys. And we will probably not include features just used by Livepeer Inc engineers, like modifying the object store db fields. I wonder if there could be a different sort of command stored in version control for safely making these sorts of DB modifications. Nothing to worry about right now, and the new object store tab can stay in  the staging/production admin for now.

File: packages/www/components/AdminObjectStoreTable/index.tsx [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+        }
+      })
+      .catch((err) => {
+        setLoadingError(`${err}`);
+        console.error(err);
+      })
+      .finally(() => setLoading(false));
+  };
+
+  const refecth = () => {
```
Comment (iameli) [inline packages/www/components/AdminObjectStoreTable/index.tsx]:
```suggestion
  const refetch = () => {
```

File: packages/www/components/AdminObjectStoreTable/index.tsx [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+    }
+    setMessage("Creating object store");
+    createObjectStore({
+      name: objectStoreName,
+      url: objectStoreUrl,
+      publicUrl: objectStorePubUrl,
+    })
+      .then(() => {
+        setMessage("Object store created");
+        refecth();
```
Comment (iameli) [inline packages/www/components/AdminObjectStoreTable/index.tsx]:
```suggestion
        refetch();
```

File: packages/www/components/AdminObjectStoreTable/index.tsx [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+      })
+      .catch((e) => setMessage(`Error: ${e}`));
+  };
+  const disableOS = (id: string, disabled: boolean) => {
+    setSelectedObjectStore(null);
+    disableObjectStore(id, disabled)
+      .then(() => {
+        refecth();
```
Comment (iameli) [inline packages/www/components/AdminObjectStoreTable/index.tsx]:
```suggestion
        refetch();
```

**Review by iameli — APPROVED** [review]
LGTM, one typo. Haven't actually booted it up myself but it's an admin page so that's okay.

I like the `useMemo` stuff for filters and whatnot. Trying to wrap my head around how we could make a more generic `useApi` hook to handle things like cursors and ordering automatically but we probably don't have enough yet to make it worth the effort.

**Comment by darkdarkdragon** [comment]
@wohlner 
> And we will probably not include features just used by Livepeer Inc engineers, like modifying the object store db fields.

Then we probably need new category of admins - 'data admin' - that will have access to such tabs.

**Comment by wohlner** [comment]
@darkdarkdragon totally agree about the data admin.

**Pull request merged**